pyproject.toml

```diff
@@ -133,7 +133,7 @@ select = [
     # pycodestyle
     "E", "W",
     # Pyflakes
-    "F",
+    "F", "FA",
```

Comment:
I know this is very minor, but it probably ought to be its own PR since it's completely unrelated to the other changes here.